src/Compilers/VisualBasic/Portable/Syntax/SyntaxFactory.vb

```diff
@@ -1068,7 +1068,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Throw New ArgumentException(VBResources.SyntaxTreeIsNotASubmission)
             End If
 
-            Dim languageVersion As LanguageVersion = options.LanguageVersion
+            'Dim languageVersion As LanguageVersion = options.LanguageVersion
```

Comment:
Note: No longer needed as we, use the `options` to also check for feature flags, not just version numbers.